### PR TITLE
update example page to correct URLs

### PR DESCRIPTION
**Pull request by jsgv**

### CODE REVIEW

File: docs/examples/simple-embed/index.html [hunk 1]
```diff
@@ -3,35 +3,33 @@
...
 </head>
 <body>
 
-  <video id="example_video_1" class="video-js vjs-default-skin" controls preload="none" width="640" height="264"
-      poster="http://video-js.zencoder.com/oceans-clip.png"
-      data-setup="{}">
-    <source src="http://video-js.zencoder.com/oceans-clip.mp4" type='video/mp4' />
-    <source src="http://video-js.zencoder.com/oceans-clip.webm" type='video/webm' />
-    <source src="http://video-js.zencoder.com/oceans-clip.ogv" type='video/ogg' />
+  <video id="example_video_1" class="video-js vjs-default-skin" controls preload="none" width="640" height="264" poster="http://vjs.zencdn.net/v/oceans.png" data-setup="{}" autoplay>
```
Comment (gkatsev) [inline docs/examples/simple-embed/index.html:25]: can you add some newlines so the line stays within ~80 characters?

File: docs/examples/simple-embed/index.html [hunk 2]
```diff
@@ -3,35 +3,33 @@
...
+  <script src="//vjs.zencdn.net/ie8/1.1.1/videojs-ie8.min.js"></script>
 
   <!-- video.js must be in the <head> for older IEs to work. -->
-  <script src="../../video.min.js"></script>
+  <script src="//vjs.zencdn.net/5.4.4/video.min.js"></script>
 
   <!-- Unless using the CDN hosted version, update the URL to the Flash SWF -->
   <script>
-    videojs.options.flash.swf = "../../video-js.swf";
+    // videojs.options.flash.swf = "path/to/video-js.swf";
```
Comment (gkatsev) [inline docs/examples/simple-embed/index.html:19]: You can use this url: `http://vjs.zencdn.net/swf/5.0/video-js.swf`

**Comment by gkatsev** [comment]
A few comments. LGTM otherwise.

**Comment by rcrooks** [comment]
Need to use the CDN URL - otherwise this will break when copied to the docs site.

**Comment by gkatsev** [comment]
This is obsolete now with our videojs 6.x work for the new docs website.

**Pull request closed without merging**